src/components/Actions.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Actions(props) {
```

Comment:
Here, there was no need to pass props since this component is not getting information from the parent component